Bio/Align/__init__.py

```diff
@@ -1022,7 +1022,6 @@ def __format__(self, format_spec):
         start1 = end1
         start2 = end2
         for end1, end2 in path[1:]:
-            gap = 0
```

Comment:
gap is redeclard in lines 1026 and 1031 so it is not necessary to declared it here

Reply:
Are you sure? What about the else clause at line 1037?

(This might mean we need another test)

Reply:
@carlosp420  is right

Reply:
Ah - I see it now, ``gap`` doesn't get used in the else clause or later. Thanks.